### PR TITLE
Upgrade libraries - app confirmed to compile and run.

### PR DESCRIPTION
**Pull request by vvasuki**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+    compileSdkVersion 27
+    buildToolsVersion '28.0.2'
 
     defaultConfig {
-        applicationId "com.ichi2.apisample"
-        minSdkVersion 15
-        targetSdkVersion 23
+        applicationId "com.github.vi.AnkiImporter"
+        // Setting minSdkVersion to 20 to enable Instant Run.
+        minSdkVersion 20
```
Comment (vi) [inline app/build.gradle:10]: So it won't work on my own device?

Is there some real reason to limit to  Android >5.0?

**Comment by vi** [comment]
Will three be also a third pull request to create some semblance of UX?

File: app/build.gradle (hunk 1 above)
Comment (vvasuki) [reply app/build.gradle:10]: Ah - nothing beyond enabling "Instant Run". You can undo that minSdkVersion change as you merge.

**Comment by vvasuki** [comment]
I want to create a UI so as to be able to import directly from the URL of some online csv.

**Comment by vi** [comment]
A usual file opener (without a hard coded directory) would also be nice.

**Pull request merged**